src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -84,6 +84,13 @@ trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDi
       }
     } ~
     path("admin" / "groups" / Segment / "members") { (groupName) => //add members to group
+      put {
+        entity(as[RawlsGroupMemberList]) { memberList =>
+          requestContext => perRequest(requestContext,
+            UserService.props(userServiceConstructor, userInfo),
+            UserService.OverwriteGroupMembers(RawlsGroupRef(RawlsGroupName(groupName)), memberList))
```

Comment:
So the rationale is PUT = "set the group members to exactly this list", POST = "add these things to the list", and DELETE = "remove these things from the list"? I'm okay with that, but document the hell out of it (the comment on 86 is misleadingly placed).